src/notification.js

```diff
@@ -34,9 +35,9 @@ export class Notification {
     this.setContainer();
     let aureliaComposedListener = () => {
       this.setContainer();
-      document.removeEventListener('aurelia-composed', aureliaComposedListener);
+      DOM.removeEventListener('aurelia-composed', aureliaComposedListener);
```

Comment:
DOM is DOM. DOMs an abstraction for document for browser compatibility. basically never use window or document, but DOM and PLATFORM(?or so)